File: CLASE II - TDA/clase_2.1_ayed/clase_1_ayed/c/main.c
#include <stdio.h>
#include <stdlib.h>
#include <time.h>

#include "persona.h"
#include "carton.h"


///Persona anidada con carton, SIN TDA

int main()
{
    srand(time(0));
    printf("------ Clase 1 - AYED - REPASO -------\n");


    PersonaPtr jugador = crearPersonaYCarton(123, "Nico", "Av. Peron 1130" );
    mostrarPersona(jugador);


    return 0;
}

File: CLASE II - TDA/clase_2.1_ayed/clase_1_ayed/golang/main.go
package main

import (
	"fmt"
	"math/rand"
	"time"
)

func main() {
	rand.Seed(time.Now().UnixNano())
	fmt.Printf("------ Clase 1 - AYED - REPASO -------\n")

	jugador := crearPersonaYCarton(123, "Nico", "Av. Peron 1130")
	mostrarPersona(jugador)
}

